esp/bindings/http/platform/httptransport.cpp

```diff
@@ -1971,6 +1971,10 @@ IFile* CHttpRequest::createUploadFile(const char * netAddress, const char* fileP
     }
     tmpFileName.appendf("%s/%s.part", filePath, fileName.str());
 
+    //If no netAddress is specified, create a local file.
+    if (isEmptyString(netAddress))
+        return createIFile(tmpFileName);
```

Comment:
Not new, but looking at the code above this..
It looks rather odd, reversing twice, strchr and writing inside StringBuffer..
What is it trying to do exactly?

it's called tmpFileName, should it be using createUniqueFile instead?

Reply:
The code may be replaced by pathTail(). Will do.